content/docs/release-notes/2305-release-notes.md

```diff
@@ -0,0 +1,187 @@
+
+## sig-content
+
+* Adds support for global python scripts per asset type to simply customizations in the scene pipeline.
+* A new Asset Browser experience has been prepared and turned on by default. The new Asset Browser experience includes three new layout modes (table view, thumbnail view, list view), file operations (add, delete, edit, rename, move), asset drag-and-drop support, URL bar for easy sharing and editing, breadcrumb and navigation bar, new detached Asset Browser inspector panel, filter options to hide unusable Editor or engine assets, and all new asset icons.
+	* To disable the new Asset Browser, disable the `ed_useWIPAssetBrowserDesign` console variable from the **Tools** menu.
+	* If the new Asset Browser layout is not properly set, restore the default layout from the **View** -> **Layouts** menu option.
+	* Searching is disabled in the table view while this feature is being developed, please use the thumbnail or list view to search for assets.
+	* Check out all the work done in the [Asset Browser project](https://github.com/orgs/o3de/projects/16) page!
+* Gems creators can now specify compatible platforms, which can be filtered in the Gem catalog.
+* Users can choose to edit existing gems `.json` files via the **Create a Gem** wizard or CLI.
+* O3DE utility widgets in the AzQtComponents library are now exposed to Python and PySide. This allows Python-based developers of tools and gems to use or extend AzQtComponents for tools, plugins, or automation.
+* Provides the ability to manage and visualize prefab overrides from O3DE Editor's Entity Outliner. Users can now override prefabs by adding or removing entities, nested prefabs, and components or changing component properties. For more information, please refer to the [Override a Prefab](https://www.o3de.org/docs/learning-guide/tutorials/entities-and-prefabs/override-a-prefab/) page.
+* Versioning Framework allows the engine to understand what version it is and adds the ability for feature, Gem, template, and project creators to specify versions that the engine can compare and potentially validate against.
+* Added ability to install multiple versions of O3DE on the same drive. Projects are aware of which O3DE version they were created in and last used with. UX adjustments to better communicate engine version to the user.
+* Added ability to download an O3DE Linux Snap package from the Snap Store and from O3DE.org binaries. Snap packaging improves installation time and reduces (in many cases eliminates) the need to download dependencies.
+* Added an optional mode to improve Asset Processor start-up time when users are simply making code changes.
```

Comment:
would ideally like to see a link to a PR so I know how to activate this optional mode.

Reply:
Added PR link.